src/admin/containers/UserResumesTable.tsx

```diff
@@ -10,13 +10,162 @@ interface UserResumesTableContainerProps {
+//     lastUpdated: '2023-03-09T23:36:23.384Z',
+//     user: {
+//       uuid: '29eeb3e2-71d2-4402-8db2-2e040f22a880',
+//       firstName: 'Ava',
+//       lastName: 'Johnson',
+//       profilePicture: null,
+//       graduationYear: 2029,
+//       major: 'Bioinformatics',
+//       bio: null,
+//       points: 0,
+//     },
+//   },
+//   {
+//     uuid: 'c9f39580-0a21-48e6-a3a0-ec4f49ef072d',
+//     isResumeVisible: true,
+//     url: 'https://acmucsd-membership-portal.s3.us-west-1.amazonaws.com/portal/resumes/29eeb3e2-71d2-4402-8db2-2e040f22a880/Jacob_Bolano_Resume.pdf',
+//     lastUpdated: '2023-03-10T23:36:23.384Z',
+//     user: {
+//       uuid: '29eeb3e2-71d2-4402-8db2-2e040f22a880',
+//       firstName: 'Ethan',
+//       lastName: 'Wong',
+//       profilePicture: null,
+//       graduationYear: 2029,
+//       major: 'Bioinformatics',
+//       bio: null,
+//       points: 0,
+//     },
+//   },
+//   {
+//     uuid: 'c9f39580-0a21-48e6-a3a0-ec4f49ef072c',
+//     isResumeVisible: true,
+//     url: 'https://acmucsd-membership-portal.s3.us-west-1.amazonaws.com/portal/resumes/29eeb3e2-71d2-4402-8db2-2e040f22a880/Jacob_Bolano_Resume.pdf',
+//     lastUpdated: '2023-03-10T23:36:23.384Z',
+//     user: {
+//       uuid: '29eeb3e2-71d2-4402-8db2-2e040f22a880',
+//       firstName: 'Jasmine',
+//       lastName: 'Patel',
+//       profilePicture: null,
+//       graduationYear: 2029,
+//       major: 'Bioinformatics',
+//       bio: null,
+//       points: 0,
+//     },
+//   },
+//   {
+//     uuid: 'c9f39580-0a21-48e6-a3a0-ec4f49ef072b',
+//     isResumeVisible: true,
+//     url: 'https://acmucsd-membership-portal.s3.us-west-1.amazonaws.com/portal/resumes/29eeb3e2-71d2-4402-8db2-2e040f22a880/Jacob_Bolano_Resume.pdf',
+//     lastUpdated: '2023-03-10T23:36:23.384Z',
+//     user: {
+//       uuid: '29eeb3e2-71d2-4402-8db2-2e040f22a880',
+//       firstName: 'Matthew',
+//       lastName: 'Lee',
+//       profilePicture: null,
+//       graduationYear: 2029,
+//       major: 'Bioinformatics',
+//       bio: null,
+//       points: 0,
+//     },
+//   },
+//   {
+//     uuid: 'c9f39580-0a21-48e6-a3a0-ec4f49ef072a',
+//     isResumeVisible: true,
+//     url: 'https://acmucsd-membership-portal.s3.us-west-1.amazonaws.com/portal/resumes/29eeb3e2-71d2-4402-8db2-2e040f22a880/Jacob_Bolano_Resume.pdf',
+//     lastUpdated: '2023-03-10T23:36:23.384Z',
+//     user: {
+//       uuid: '29eeb3e2-71d2-4402-8db2-2e040f22a880',
+//       firstName: 'Olivia',
+//       lastName: 'Rivera',
+//       profilePicture: null,
+//       graduationYear: 2029,
+//       major: 'Bioinformatics',
+//       bio: null,
+//       points: 0,
+//     },
+//   },
+// ];
+
 const UserResumesTableContainer: React.FC<UserResumesTableContainerProps> = (props) => {
   const { resumes, getAllVisibleResumes } = props;
 
   useEffect(() => {
     getAllVisibleResumes();
   }, [getAllVisibleResumes]);
 
+  // Can set resumes to testingData to use testingData
```

Comment:
We can delete all of the comments in this file and unused code. We just need comments when we're documenting a hard to read portion of code or function for other developers to maintain in the future.

Reply:
Bump